haros/haros.py

```diff
@@ -317,10 +319,14 @@ def main(argv = None, source_runner = False):
                             level = logging.DEBUG)
     else:
         logging.basicConfig(level = logging.WARNING)
+
     try:
+        if args.dir:
+            os.chdir(args.dir)
```

Comment:
I'm not sure actually what the behaviour is across platforms. If `chdir` is completely local to the `haros` process (ie: `python`) then there is of course no need to restore it.